src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -104,6 +121,17 @@ export const TrackPanelBookmarks = (props: TrackPanelBookmarksProps) => {
 
   const limitedPreviouslyViewedObjects = previouslyViewedObjects.slice(-20);
 
+  const onEllipsisClick = () => {
+    analyticsTracking.trackEvent({
+      category: 'recent_bookmark_link',
+      label: 'recent_bookmarks',
+      action: 'clicked',
+      value: previouslyViewedObjects.length
+    });
```

Comment:
Either the code or the google doc is wrong about the value of the category field; need to be put in agreement with each other:

![image](https://user-images.githubusercontent.com/6834224/68431831-0b5e8280-01ab-11ea-9750-aacc10eae5de.png)

Reply:
That's done. Thanks 👍